test_runner/regress/test_storage_controller.py

```diff
@@ -935,7 +935,7 @@ def test_storage_controller_debug_apis(neon_env_builder: NeonEnvBuilder):
     that just hits the endpoints to check that they don't bitrot.
     """
 
-    neon_env_builder.num_pageservers = 2
+    neon_env_builder.num_pageservers = 3
```

Comment:
3 pageservers because we need some node that contains neither an attached nor secondary location, in order to migrate a secondary there